content/release/2.0.0.md

```diff
@@ -0,0 +1,442 @@
+---
+title: Release 2.0.0 available
+date: 2025-03-26
```

Comment:
Release date is usually the end of vote or afterwards, if the announcement comes with delay.

Reply:
Updated. 2025-04-28. I plan to announce it officially later tonight.